target/scripts/startup/setup-stack.sh

```diff
@@ -67,20 +67,20 @@ function _setup_default_vars
   PFLOGSUMM_RECIPIENT="${PFLOGSUMM_RECIPIENT:=${REPORT_RECIPIENT}}"
   LOGWATCH_RECIPIENT="${LOGWATCH_RECIPIENT:=${REPORT_RECIPIENT}}"
 
+  VARS[LOGWATCH_RECIPIENT]="${LOGWATCH_RECIPIENT}"
+  VARS[PFLOGSUMM_RECIPIENT]="${PFLOGSUMM_RECIPIENT}"
+  VARS[PFLOGSUMM_SENDER]="${PFLOGSUMM_SENDER}"
+  VARS[PFLOGSUMM_TRIGGER]="${PFLOGSUMM_TRIGGER}"
+  VARS[POSTMASTER_ADDRESS]="${POSTMASTER_ADDRESS}"
+  VARS[REPORT_RECIPIENT]="${REPORT_RECIPIENT}"
+  VARS[REPORT_SENDER]="${REPORT_SENDER}"
+
   local VAR
   for VAR in "${!VARS[@]}"
   do
-    echo "export ${VAR}='${VARS[${VAR}]}'" >>/root/.bashrc
+    echo "export ${VAR}='${VARS[${VAR}]}'" >/root/.bashrc
+    echo "${VAR}='${VARS[${VAR}]}'"        >/etc/dms-settings
```

Comment:
Should be

``` suggestion
    echo "export ${VAR}='${VARS[${VAR}]}'" >>/root/.bashrc
    echo "${VAR}='${VARS[${VAR}]}'"        >>/etc/dms-settings
```

Maybe then even clear `/etc/dms-settings` before the loop runs? (`: >/etc/dms-settings`)

Reply:
In a fresh container, `.bashrc` does only contain some comments.

I chose to overwrite the file, to avoid possible duplicate entrys, when for example, the container is restarted.

Reply:
Holy shit. That makes no sense. You are absolutly right.